articles/ai-services/computer-vision/includes/identity-curl-quickstart.md

```diff
@@ -32,58 +32,134 @@ Get started with facial recognition using the Face REST API. The Face service pr
 > [!NOTE]
 > If you haven't received access to the Face service using the [intake form](https://aka.ms/facerecognition), some of these functions won't work.
 
-1. First, call the Detect API on the source face. This is the face that we'll try to identify from the larger group. Copy the following command to a text editor, insert your own key, and then copy it into a shell window and run it.
+1. First, call the Detect API on the source face. This is the face that we'll try to identify from the larger group. Copy the following command to a text editor, insert your own key and endpoint, and then copy it into a shell window and run it.
+
+    #### [Windows](#tab/windows)
```

Comment:
Let's name these tabs PowerShell and bash

Reply:
The reason why I named the tabs "windows/linux" is to sync with other frames hosted under same URI. For example, if a reader already select "linux" in C# frame when setting up environment vars, he'll see Bash commands automatically once he navigates to REST frame, instead of PowerShell.

Besides, the code section already has the ability to show the written language. This is a sample preview of what I expect:

![Imgur](https://i.imgur.com/6LvMXXl.png)

Any thoughts? I'm also good with naming them PowerShell/Bash directly, to be clear.